Podfile

```diff
@@ -27,15 +27,15 @@ def aztec
 end
 
 def tracks
-  pod 'Automattic-Tracks-iOS', '~> 2.0.0-beta.1'
+  pod 'Automattic-Tracks-iOS', '~> 2.0.0'
   # pod 'Automattic-Tracks-iOS', :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :branch => 'trunk'
   # pod 'Automattic-Tracks-iOS', :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :commit => ''
   # pod 'Automattic-Tracks-iOS', :path => '../Automattic-Tracks-iOS'
 end
 
 def wordpress_kit
   # To allow pod to pick up beta versions use -beta. E.g., 1.1.7-beta.1
-  pod 'WordPressKit', '~> 6.0-beta'
+  pod 'WordPressKit', '~> 6.0'
```

Comment:
I think our existing logic in release-toolkit warns if there's a `-beta` prefix in the version rule, but it should be noted that leaving it there doesn't hurt.

Running `bundle exec pod update WordPressKit` with the rule `~> 6.0-beta`, for example, would have resulted in version 6.0.0 being installed.

Reply:
E.g., this is what was run during the 21.7 code freeze:

![image](https://user-images.githubusercontent.com/1218433/216869666-54760bcc-b1d4-4b2d-a59d-dab935fed613.png)

Reply:
I'm not necessarily arguing for keeping the `-beta` prefix. Just wanted to point out this noteworthy CocoaPods behavior.

Reply:
Gotcha @mokagio, thanks for explaining! Do you usually only update the `Podfile` when it's necessary based on the version rule?